c_glib/test/test-record-batch.rb

```diff
@@ -253,7 +254,10 @@ def test_valid
       def test_invalid
         message = "[record-batch][validate-full]: Invalid: " +
           "In column 1: Invalid: Invalid UTF8 sequence at string index 0"
-        record_batch = Arrow::RecordBatch.new(@schema, @n_rows, [@uint8_value, @invalid_name_value])
+        record_batch = Arrow::RecordBatch.new(@schema, @n_rows,
+                                              [@uint8_value,
+                                               @invalid_name_value]
+                                             )
```

Comment:
Let's create a variable:

```suggestion
        columns = [@uint8_value, @invalid_name_value]
        record_batch = Arrow::RecordBatch.new(@schema, @n_rows, columns)
```